src/main/kotlin/icons/SnykIcons.kt

```diff
@@ -25,6 +25,7 @@ object SnykIcons {
     val CONTAINER = getIcon("/icons/container.svg", SnykIcons::class.java)
     val CONTAINER_DISABLED = getIcon("/icons/container_disabled.svg", SnykIcons::class.java)
     val CONTAINER_IMAGE = getIcon("/icons/container_image.svg", SnykIcons::class.java)
+    val CONTAINER_IMAGE_24 = getIcon("/icons/container_image_24.svg", SnykIcons::class.java)
```

Comment:
@ArtsiomCh, can we scale the icon ('cause it's SVG)?

Reply:
Didn't find an easy way to do that. Looks like separate svg with `width="24" height="24"` is an easiest solution here. Any advice welcome.

Reply:
Yeah, usually scaling doesn't really work due to sub-pixels and such. Having separate icons improves UX